### DISCUSSION TITLE
In local script energy scheduling, batteries cannot sell power to the grid

### DISCUSSION THREAD

**Discussion by 18140023670** [NREL/REopt-API-Analysis#158] [7 - Technology Specific Questions] [answered]
Hello, I found a doubt in the process of using REopt script to calculate home energy scheduling, that is, there is no item sent to the grid in the battery output (ElectricStorage). May I ask if the REopt local script does not support selling electricity from the battery to the grid? If so, how do I configure the battery to sell electricity to the grid? Looking forward to your reply, thanks！

**Comment by adfarth** [comment, marked as answer]
Hello, it is currently not possible to allow ElectricStorage to export to the grid in REopt. Re-configuring the underlying [REopt.jl](https://github.com/NREL/REopt.jl) code to allow for this would require many modifications to the constraints and calculations (especially the emissions calculations). However, it is noted that this would be a helpful feature to include in REopt in the future!

**Comment by 18140023670** [comment]
Thank you very much for your reply. Does REopt plan when to realize the function of battery to power grid?
